pandas/core/indexes/category.py

```diff
@@ -32,27 +32,65 @@
 
 class CategoricalIndex(Index, accessor.PandasDelegate):
     """
+    Immutable Index implementing an ordered, sliceable set.
 
-    Immutable Index implementing an ordered, sliceable set. CategoricalIndex
-    represents a sparsely populated Index with an underlying Categorical.
+    CategoricalIndex represents a sparsely populated Index with an underlying
+    Categorical.
 
     Parameters
     ----------
     data : array-like or Categorical, (1-dimensional)
     categories : optional, array-like
-        categories for the CategoricalIndex
+        Categories for the CategoricalIndex.
     ordered : boolean,
-        designating if the categories are ordered
+        Designating if the categories are ordered.
     copy : bool
-        Make a copy of input ndarray
+        Make a copy of input ndarray.
     name : object
-        Name to be stored in the index
+        Name to be stored in the index.
 
     Attributes
     ----------
-    codes
-    categories
-    ordered
+    codes : ndarray
+        The codes (integer positions, which point to the categories) of this
+        CategoricalIndex, read only.
+    categories : Index
+        The categories of this CategoricalIndex.
+    ordered : boolean
+        Whether or not this CategoricalIndex is ordered. An ordered
+        CategoricalIndex allows to have .min() and .max() values.
```

Comment:
Not something you could know (sphinx detail related to our docs), but: you don't need to include this, as the html generation will automatically turn this into a table with those explanations added.